src/Console/Output/ProcessCommandReporter.php

```diff
@@ -43,7 +43,11 @@ public function reportLoadedRectors(): void
      */
     public function reportChangedFiles(array $changedFiles): void
     {
-        $this->symfonyStyle->title(sprintf('%s Changed Files', count($changedFiles)));
+        $this->symfonyStyle->title(sprintf(
+            '%s Changed File%s',
```

Comment:
Change **f**iles

Reply:
Hm, why?
As we are reporting the _changed_ files, in the past, we should use the verb in past. `x Change files` would sound like: You have X file of change, and not, You have X changed files :smile: